docs/flow/modin/core/execution/unidist/implementations/pandas_on_unidist/index.rst

```diff
@@ -6,7 +6,8 @@ PandasOnUnidist Execution
 Queries that perform data transformation, data ingress or data egress using the `pandas on Unidist` execution
 pass through the Modin components detailed below.
 
-To enable `pandas on Unidist` execution, please refer to the usage section in :doc:`pandas on Unidist </development/using_pandas_on_unidist>`.
+To enable `pandas on MPI through unidist` execution,
```

Comment:
Such changes in documentation can be made independently from the new release of the unidist, right?

Reply:
Right. unidist 0.5.0 is out so it is okay anyway.